### PR TITLE
docs: change references slack

### LINKED ISSUE #5090: [DOCS] Change references slack

### PR DESCRIPTION
**Pull request by sdiazlor**
# Pull Request Template

Closes #5090

* READMEs badges and references
* Contributing docs
* Other doc references
* GitHub discussions

**Type of change**

- Documentation update

**How Has This Been Tested**

sphinx-autobuild

PHOTO-2024-06-24-14-42-40

mkdocs-serve
Screenshot 2024-06-24 at 16 42 18

**Checklist**

- I made corresponding changes to the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ However you choose to contribute, please be mindful and respect our
 
 ## Need some help?
 
-We understand that getting started might be scary and difficult, therefore, we provide a guided contributor mentorship program. Feel free to schedule a meeting [here](https://calendly.com/david-berenstein-huggingface/30min) to get you started. Alternatively, you can start with [joining our Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) and reaching out in our [contributor channel](https://rubrixworkspace.slack.com/archives/C05BCGM277B).
+We understand that getting started might be scary and difficult, therefore, you can join our discord channel [#argilla-general](https://discord.gg/hugging-face-879548962464493619) by selecting *Argilla* in *Channels and Roles*.
```
Comment (davidberenstein1957) [inline CONTRIBUTING.md]: I would avoid using the word "scary" 
```suggestion
We understand that getting started might be a bit difficult, therefore, you can join our discord channel [#argilla-general](https://discord.gg/hugging-face-879548962464493619) by selecting *Argilla* in *Channels and Roles* after joining the Discord.
```

File: argilla/README.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ We are an open-source community-driven project and we love to hear from you. Her
 
 - [Community Meetup](https://lu.ma/embed-checkout/evt-IQtRiSuXZCIW6FB): listen in or present during one of our bi-weekly events.
 
-- [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g): get direct support from the community.
+- [Discord](http://hf.co/join/discord): get direct support from the community.
```
Comment (davidberenstein1957) [inline argilla/README.md]: I would mention the channels they need to join.

File: argilla-v1/README.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ We are an open-source community-driven project and we love to hear from you. Her
 
 - [Community Meetup](https://lu.ma/embed-checkout/evt-IQtRiSuXZCIW6FB): listen in or present during one of our bi-weekly events.
 
-- [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g): get direct support from the community.
+- [Discord](https://discord.gg/hugging-face-879548962464493619"): get direct support from the community.
```
Comment (davidberenstein1957) [inline argilla-v1/README.md]: mention channels

**Comment by davidberenstein1957** [comment]
@sdiazlor , the Join Discord buttons look a bit weird to me. Would it be possible to just dhow the discord logo? or only a square logo instead of the entire text?

**Comment by sdiazlor** [comment]
Captura de pantalla 2024-06-26 004250

@davidberenstein1957 WDYT? Writing Discord directly and using only the icon logo I believe is more consistent with the rest of buttons

**Comment by davidberenstein1957** [comment]
Looks great!

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**